rebasehelper/application.py

```diff
@@ -708,7 +717,7 @@ def run(self):
                     Application.check_build_requires(self.spec_file)
                 # Build packages
                 try:
-                    build = self.build_packages()
+                    build = self.build_packages
```

Comment:
So this is because you changed the function to a property, which actually does not make any sense. Please revert it.